CMakeLists.txt

```diff
@@ -82,6 +83,10 @@ add_definitions(
     -DQT_MESSAGELOGCONTEXT #enable function name and line number in debug output
 )
 
+if(QUICK_COMPILER)
```

Comment:
do not you need to define the option before checking for it (cannot remember the exact `cmake` behavior here)

Reply:
We actually don't need this at all, as the `QUICK_COMPILER` isn't used in our C++ code. Removed it